scripts/run_imitation_server.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
```

Comment:
Same here: Leftover / why necessary?

Reply:
This line of code has been copied from (https://github.com/patrickab/prefq/blob/main/scripts/run_imitation_example.sh) without further consideration

Reply:
I think those changes should not be relevant for the imitation integration.